dev/provider_packages/prepare_provider_packages.py

```diff
@@ -2117,6 +2117,7 @@ def summarise_total_vs_bad_and_warnings(total: int, bad: int, warns: List[warnin
     ("SelectableGroups dict interface is deprecated. Use select.", "kombu"),
     ("The module cloudant is now deprecated. The replacement is ibmcloudant.", "cloudant"),
     ("distutils Version classes are deprecated. Use packaging.version instead.", "dask"),
+    ("distutils Version classes are deprecated. Use packaging.version instead.", "fsspec"),
```

Comment:
In the mean time https://github.com/apache/airflow/pull/20444 was merged
so can you please remove this change and rebase?

Reply:
Yes ofc, done.